src/app/modules/display.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :+1:

Reply:
Okay, I'll correct it. Thank you for your suggestions and for reviewing my project.